src/common/paths/delete.yml

```diff
@@ -128,6 +139,9 @@ post:
         #### InfluxDB OSS
 
         - Deleted the data.
+      content:
```

Comment:
204 Successful Response was not showing before, so I added an empty response since that is technically what is being returned

Reply:
> 

@sunbryely-influxdata Researched this a bit and didn't any good solution for displaying '204' in the response preview pane while also keeping the response content-free. Seems to be a common problem/wish across other OpenAPI tools. Rapidoc appears to behave similarly.

Once we beef up the spec, then we can replace Redoc and build the UI behavior that we want.